src/services/app.services.ts

```
@@ -27,7 +27,12 @@ export const createTodo = async (
       title,
       completed
     });
-    return { data: response.data, error: null };
+	const a= "hida";
+	a = 20; // This line will cause a TypeScript error because 'a' is a string and cannot be assigned a number
```

Biome (1.9.4)

[error] Can't assign a because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)